src/core/backend/pearl/holder.rs

```diff
@@ -148,17 +148,17 @@ impl Holder {
         if state.is_ready() {
             let storage = state.get();
             trace!("Vdisk: {}, read key: {}", self.vdisk, key);
-            PEARL_GET_COUNTER.count(1);
-            let timer = PEARL_GET_TIMER.start();
+            counter!(PEARL_GET_COUNTER, 1);
+            let timer = Instant::now();
             storage
                 .read(Key::from(key))
                 .await
                 .map(|r| {
-                    PEARL_GET_TIMER.stop(timer);
+                    counter!(PEARL_GET_TIMER, timer.elapsed().as_nanos() as u64);
```

Comment:
We need to count all get requests. Here you count only Ok results

Reply:
I didn't touch this part of code, but it seems that it's bug, I fixed that.